tests/integration/test_vulnerability_detector/test_feeds/alas/test_extra_tags_alas_feed.py

```diff
@@ -0,0 +1,110 @@
+custom_alas_json_feed_path = os.path.join(test_data_path, 'feeds', vd.CUSTOM_ALAS_JSON_FEED)
+custom_alas2_json_feed_path = os.path.join(test_data_path, 'feeds', vd.CUSTOM_ALAS2_JSON_FEED)
+
+wazuh_log_monitor = FileMonitor(LOG_FILE_PATH)
+
+# Set Amazon Linux 1 and 2 configuration
+# Set Amazon Linux 1 and 2 configuration
+parameters = [
+    {'ALAS_CUSTOM_FEED': custom_alas_json_feed_path, 'OS': 'amazon-linux'},
+    {'ALAS_CUSTOM_FEED': custom_alas2_json_feed_path, 'OS': 'amazon-linux-2'}
+]
+metadata = lower_case_key_dictionary_array(parameters)
+ids = [
+    'ALAS_configuration',
+    'ALAS2_configuration'
+]
+
+
+# Configuration data
+configurations = load_wazuh_configurations(configurations_path, __name__, params=parameters, metadata=metadata)
+
+
+@pytest.fixture(scope='module', params=configurations, ids=ids)
+def get_configuration(request):
+    """Get configurations from the module."""
+    return request.param
+
+
+@pytest.fixture
+def modify_feed(test_values, get_configuration, request):
+    """Modify the Amazon Linux JSON feed by setting a test tag value."""
+    alas_feed_path = get_configuration['metadata']['alas_custom_feed']
+    if alas_feed_path == custom_alas_json_feed_path:
+        custom_feed_path = custom_alas_json_feed_path
+    elif alas_feed_path == custom_alas2_json_feed_path:
+        custom_feed_path = custom_alas2_json_feed_path
+
+    backup_data = read_json_file(custom_feed_path)
+    modified_data = deepcopy(backup_data)
+
+    # Insert key:value pair as string, since otherwise, you could not insert lists or dictionaries as a key
+    modified_string_data = vd.insert_data_json_feed(data=modified_data, field_name=test_values[0],
+                                                    field_value=test_values[1], append_data=None, brackets=False)
+
+    write_file(custom_feed_path, modified_string_data)
+
+    vd.clean_vuln_and_sys_programs_tables()
+
+    control_service('restart', daemon='wazuh-modulesd')
+
+    vd.set_system(system='ALAS')
+
+    yield
+
+    write_json_file(custom_feed_path, backup_data)
+
+    vd.clean_vuln_and_sys_programs_tables()
+
+    truncate_file(LOG_FILE_PATH)
+
+
+def test_no_feed_changes(clean_vuln_tables, get_configuration, configure_environment, restart_modulesd):
+    """Check if the feed is imported successfully by default."""
+    if get_configuration['metadata']['alas_custom_feed'] == custom_alas_json_feed_path:
+        vd.check_feed_imported_successfully(wazuh_log_monitor=wazuh_log_monitor, log_system_name=vd.ALAS_LOG,
+                                            expected_vulnerabilities_number=vd.ALAS_NUM_CUSTOM_VULNERABILITIES)
+    elif get_configuration['metadata']['alas_custom_feed'] == custom_alas2_json_feed_path:
+        vd.check_feed_imported_successfully(wazuh_log_monitor=wazuh_log_monitor, log_system_name=vd.ALAS2_LOG,
+                                            expected_vulnerabilities_number=vd.ALAS2_NUM_CUSTOM_VULNERABILITIES)
```

Comment:
In order to improve the speed of all VD tests we should include these fixtures in VD contest:
```
@pytest.fixture(scope='function')
def restart_modulesd_function():

    control_service('restart', daemon='wazuh-modulesd')

    yield 

    control_service('stop', daemon='wazuh-modulesd')


@pytest.fixture(scope='module')
def set_system():
    vd.set_system(system='ALAS')


@pytest.fixture(scope='module')
def up_required_vd_daemons():

    required_vd_daemons = ['wazuh-db', 'wazuh-analysisd', 'wazuh-modulesd']

    for daemon in required_vd_daemons:
        control_service('start', daemon)
    
    yield

    for daemon in required_vd_daemons:
        control_service('stop', daemon)
```

Using these fixtures the time used for this module reduce from 45 minutes to 15 

```suggestion
def test_no_feed_changes(up_required_vd_daemons, clean_vuln_tables, clean_vuln_tables_function,  get_configuration, configure_environment, daemons_handler, 
                         file_monitoring):
    """Check if the feed is imported successfully by default."""
    if get_configuration['metadata']['alas_custom_feed'] == custom_alas_json_feed_path:
        vd.check_feed_imported_successfully(wazuh_log_monitor=wazuh_log_monitor, log_system_name=vd.ALAS_LOG,
                                            expected_vulnerabilities_number=vd.ALAS_NUM_CUSTOM_VULNERABILITIES)
    elif get_configuration['metadata']['alas_custom_feed'] == custom_alas2_json_feed_path:
        vd.check_feed_imported_successfully(wazuh_log_monitor=wazuh_log_monitor, log_system_name=vd.ALAS2_LOG,
                                            expected_vulnerabilities_number=vd.ALAS2_NUM_CUSTOM_VULNERABILITIES)
```

Reply:
Done in commit 330f641b6fd3cb999e7d4142cfe95ec1854f80ef and 869101f5e36f1c088568d9b5d3f5e66c465768f4